js/chirp.js

```diff
@@ -1,7 +1,14 @@
-function chirp(n){
+function chirp(n) {
   // FIX ME
+  let bird = "chirp ";
+
+  if (n === 1) {
```

Comment:
There's a way to simplify this even more by making sure that only the recursed functions add a space, then in your base case, you'll just return an empty string.